governance/scripts/deployments/emptyProxy.js

```diff
@@ -0,0 +1,42 @@
+const { ethers } = require('hardhat')
```

Comment:
Please rename that file to `emptyProxyAndUpSwapImplementation.js`

Reply:
ok done